### PR TITLE
fix make release-android command

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

**Review by yenda — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -46,6 +46,7 @@ prepare-ios: prepare ##@prepare Install iOS specific dependencies
 release: release-android release-ios ##@build build release for Android and iOS
 
 release-android: prod-build-android ##@build build release for Android
+	cd android; ./gradlew react-native-android:installArchives
```
Comment (jeluard) [inline Makefile:49]: Does this need to be done each time? How about using the setup script?

Comment (flexsurfer) [reply Makefile:49]: yeah had a same question here

Comment (rasom) [reply Makefile:49]: @jeluard i'm not familiar with makefile's syntax/structure and have no idea where that setup script lives, besides i'm not using `make`. So if you know the way to make it better please add commit to this PR

Comment (jeluard) [reply Makefile:49]: I am not familiar with make either but probably you don't want to install this frequently.
Setup script is here https://github.com/status-im/status-react/blob/develop/scripts/setup
Please update it accordingly

**Review by jeluard — CHANGES_REQUESTED** [review]
This change is better suited in the setup script

**Pull request closed without merging**